Source/WebKit/webpushd/_WKMockUserNotificationCenter.mm

```diff
@@ -50,31 +54,48 @@
 @implementation _WKMockUserNotificationCenter {
     RetainPtr<NSString> m_bundleIdentifier;
     RetainPtr<NSMutableArray> m_notifications;
+    RetainPtr<NSNumber> m_appBadge;
 }
 
-- (instancetype)initWithBundleIdentifier:(NSString *)bundleIdentifier
+- (instancetype)_internalInitWithBundleIdentifier:(NSString *)bundleIdentifier
 {
     self = [super init];
     if (!self)
         return nil;
 
     m_bundleIdentifier = bundleIdentifier;
-    m_notifications = notificationsByBundleIdentifier(bundleIdentifier);
+    m_notifications = adoptNS([[NSMutableArray alloc] init]);
+
     return self;
 }
 
+- (instancetype)initWithBundleIdentifier:(NSString *)bundleIdentifier
+{
+    self = centersByBundleIdentifier(bundleIdentifier);
```

Comment:
I guess in theory you have to release self here before reassigning it, not sure if we care for a mock.